src/dmd/dscope.d

```diff
@@ -209,21 +209,11 @@ struct Scope
     extern (C++) Scope* pop()
     {
         //printf("Scope::pop() %p nofree = %d\n", this, nofree);
-        Scope* enc = enclosing;
         if (enclosing)
-        {
-            enclosing.ctorflow.callSuper |= ctorflow.callSuper;
-            if (ctorflow.fieldinit.length)
-            {
-                if (enclosing.ctorflow.fieldinit.length)
-                {
-                    assert(ctorflow.fieldinit.ptr != enclosing.ctorflow.fieldinit.ptr);
-                    foreach (i, u; ctorflow.fieldinit)
-                        enclosing.ctorflow.fieldinit[i] |= u;
-                }
-                ctorflow.freeFieldinit();
-            }
-        }
+            enclosing.ctorflow.OR(ctorflow);
```

Comment:
Indentention looks inconsistent.

Reply:
It isn't. It looks inconsistent with the deleted lines

Reply:
Ok, I’m a bit limited here on my phone.